packages/@aws-cdk/aws-appmesh/lib/private/utils.ts

```diff
@@ -92,3 +95,40 @@ export function renderMeshOwner(resourceAccount: string, meshAccount: string) :
     ? meshAccount
     : undefined;
 }
+
+/**
+ * This is the helper method to check if properties for passed match are undefined.
+ * This returns true if all properties for an object are undefined.
+ */
+export function areMatchPropertiesUndefined(match: HttpRouteMatch | GrpcRouteMatch | GrpcGatewayRouteMatch | undefined): boolean {
```

Comment:
1. No `HttpGatewayRouteMatch` here?
2. So, if I now specify `rewriteRequestHostname` as `true`, which is the same as the default, in `HttpGatewayRouteMatch`, now the default behavior will _not_ kick in for matching on the path? Is this correct?

Reply:
Removed this function.